dev/reference/sql/functions-and-operators/window-functions.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 窗口函数
+category: reference
+aliases: ['/docs-cn/sql/window-functions/']
```

Comment:
I think this is a newly added file. If yes, it's unnecessary to add `aliases` and please remove it. Otherwise, it might cause an error at the website.